pilot/pkg/security/authn/v1beta1/policy_applier_test.go

```diff
@@ -58,6 +60,69 @@ func TestJwtFilter(t *testing.T) {
 			},
 			expected: nil,
 		},
+		{
+			name: "Fallback to alpha with no JWT",
+			alphaPolicy: &authn_v1alpha1.Policy{
+				Peers: []*authn_v1alpha1.PeerAuthenticationMethod{{
+					Params: &authn_v1alpha1.PeerAuthenticationMethod_Mtls{&authn_v1alpha1.MutualTls{}},
+				}},
+			},
+			in:       []*model.Config{},
+			expected: nil,
+		},
+		{
+			name: "Fallback to alpha JWT",
+			alphaPolicy: &authn_v1alpha1.Policy{
+				Peers: []*authn_v1alpha1.PeerAuthenticationMethod{{
+					Params: &authn_v1alpha1.PeerAuthenticationMethod_Mtls{&authn_v1alpha1.MutualTls{}},
```

Comment:
why empty config reuslting into peer params mtls configured? this should be from alpha policy behavior, but still unclear to me why

Reply:
This is the input. I just want to log the case when we have (old) mTLS policy. Together with your PR, we will need to set story correct for migration.